### DISCUSSION TITLE
Wont start on macOS: 'Could not load the Qt platform plugin "cocoa" in "" even though it was found.'

### DISCUSSION THREAD

**Discussion by martin-braun** [audacious-media-player/audacious#82] [Q&A]
I installed the player via `brew install audacious`, but once I try to run it, I get:

> qt.qpa.plugin: Could not load the Qt platform plugin "cocoa" in "" even though it was found.

How can I fix this problem. I have Qt installed through brew already.

**Comment by radioactiveman** [comment]
Which Qt version did you install? Do you have multiple Qt versions installed?
You may need to set `QT_QPA_PLATFORM_PLUGIN_PATH`, see https://stackoverflow.com/a/55322249 for more details.

If it doesn't help, please report this directly to Homebrew. https://github.com/orgs/Homebrew/discussions

**Reply by martin-braun** [reply]
Shouldn't the right Qt version be installed automatically through Homebrew? At least that's what I experience in conventional software installed through Homebrew. 

Anyway I just `brew install qt` and now I have `6.6.1` installed. I then added `QT_QPA_PLATFORM_PLUGIN_PATH="$(\ls -d /usr/local/Cellar/qt/* | tail -n 1)/share/qt/plugins"
` to my `.zshrc` which nicely resolves to `/usr/local/Cellar/qt/6.6.1/share/qt/plugins`.

It still doesn't work, because there is no `cocoa` directory in the `plugins` directory. I also have `opencv-python-headless` and `opencv` installed, it doesn't satisfy `audacious`.

EDIT: `find /usr/local/Cellar/qt -iname cocoa` => nothing, `ls $HOME/Library/Qt` => no such file or directory, I need to install https://doc.qt.io/qt-6/qtgui-attribution-cocoa-platform-plugin.html, but I find no useful information due to similar issues in regards of opencv and their python bindings that clutter results in Google.

I reported it here: https://github.com/Homebrew/homebrew-core/issues/158675
